### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *Fix an issue when you try to import wallabag articles #55*
- *Remove ini file for php #57*
- *Upgrade to 2.3.8 #71*
- *Wallabag is not protected by fail2ban against brute force #65*
- *Global upgrade of the package #67*
- *Cache files are included in the backup (useless space usage) #60*

## Solution
- *Add fail2ban config*
- *Don't save the cache during backup (delete it) #71*
## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : @lapineige 
- [x] **Approval (LGTM)** : Kay0u
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/wallabag2_ynh%20PR70/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by lapineige** [comment]
What a big testing, since august :smile:

Should we wait for #69 too ?

**Comment by maniackcrudelis** [comment]
If #69 is merged before the 7 usual days, we could include it, otherwise I don't think we should "wait" for it.

**Comment by lapineige** [comment]
I already have some positive feedbacks.
My test went well too.

File: scripts/install [hunk 1]
```diff
@@ -30,130 +31,163 @@ admin=$YNH_APP_ARG_ADMIN
...
-# Initialize database
-ynh_mysql_setup_db "$db_user" "$db_name"
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
+ynh_script_progression --message="Setting up source files..." --weight=6
+
+### `ynh_setup_source` is used to install an app from a zip or tar.gz file,
+### downloaded from an upstream source, like a git repository.
+### `ynh_setup_source` use the file conf/app.src
```
Comment (kay0u) [inline scripts/install]: Please remove these lines

(Impossible to commit with my phone right now /o\)

Comment (lapineige) [reply scripts/install]: Done

Comment (kay0u) [reply scripts/install]: Thx

**Comment by lapineige** [comment]
Upgrade from previous version works fine.
Fresh install too.
I didn't test the backup+restore yet, but the CI is passing.

**Comment by lapineige** [comment]
LGTM & code review.

Let's merge in 3 days ?

**Comment by kay0u** [comment]
There's still one *LGTM* missing from the apps group

**Comment by JimboJoe** [comment]
Can be merged in 3 days.

**Pull request merged**